pkg/ddevapp/config.go

```diff
@@ -94,21 +94,41 @@ func (c *Config) Write() error {
 	return nil
 }
 
-// Read app configuration from a specified location on disk.
+// Read app configuration from a specified location on disk, falling back to defaults for config
+// values not defined in the read config file.
 func (c *Config) Read() error {
 	source, err := ioutil.ReadFile(c.ConfigPath)
 	if err != nil {
 		return err
 	}
 
+	// Read config values from file.
 	err = yaml.Unmarshal(source, c)
 	if err != nil {
 		return err
 	}
 
 	log.WithFields(log.Fields{
-		"Existing config": awsutil.Prettify(c),
+		"Read config": awsutil.Prettify(c),
 	}).Debug("Finished config read")
+
+	// If any of these values aren't defined in the config file, set them to defaults.
```

Comment:
Thanks for the simple code here. Even though the reflection code was more elegant, this is more obvious for this simple case. We'll want reflection to simplify a really complex case for sure.